webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/queries/CmdbQuery.scala

```diff
@@ -222,6 +222,31 @@ final case object NodeStateComparator extends NodeCriterionType {
 }
 
 
+final case object NodeStringComparator extends NodeCriterionType {
```

Comment:
I should make it a case class, with a parameter telling on which attribute of nodeinfo we should search

Reply:
However, the big trouble here is that it breaks the detection on which group a *pending* node will appear, as the pending node is not in cache

Reply:
and properties are not correctly used to detect in which group the node will be